source/_meta/_00-head.mustache

```diff
@@ -4,14 +4,13 @@
     <title>{{ title }}</title>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width" />
-    
+    <script src="https://use.fontawesome.com/a9bf041a16.js"></script>
     <link rel="stylesheet" href="../../css/style.css?{{ cacheBuster }}" media="all" />

```

Comment:
I think this might actually be okay as is. When Pattern Lab builds and serves from /public the directories are all correct. If I make the change described here, my styles don't load. Let's take a look together on Monday.